openshift/CLOUD-2262.adoc

```diff
@@ -258,11 +258,17 @@ This should be considered as a new feature request.
 
 === Hard Requirements
 
-* EJB remote calls have to work when application is deployed on JBoss EAP on OpenShift
-* The transaction consistency has to be guaranteed for the transactions propagated
-  over the EJB remote calls
+* allow users to deploy clustered transactional EAP applications on OpenShift
+* applications should be able to communicate using ejb-client/remoting libraries
+* distributed transactional operations among those applications should be fully supported
+* transactions should recover properly if the transaction is interrupted
+* users would create the applications using provided template, which would hide the complexity associated with operation in cloud
+* users would be able to configure connections between applications by configuring remoting subsystem in standalone-openshift.xml
```

Comment:
will we define the ways how the remote connection can be created? will we be supporting only remote outbound connection at this time?
As we discussed the recovery with the programmatic lookup does not work currently. Do you expect that being nice to have or hard requirement?

Reply:
I added programmatic lookup as 'nice-to-have' cause we don't know the reason for the problems yet.